### PR TITLE
Homework.streams.part1.exercise2(abramov)

### PR DESCRIPTION
**Pull request by german1um**

### CODE REVIEW

**Review by lesscau — APPROVED** [review]

File: src/test/java/lambda/part1/exercise/Exercise2.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+     */
+    @Test
+    void calcTotalSalaryWithCoefficientWorkExperience() {
+        List<Employee> employees = getEmployees();
+
+        Double expected = employees.stream()
+                .map(Employee::getJobHistory)
+                .map(jobs -> jobs.get(jobs.size() - 1))
+                .map(JobHistoryEntry::getDuration)
+                .map(duration -> duration > 3 ? 75000d : 75000 * 1.2)
```
Comment (vkovrovEPAM) [inline src/test/java/lambda/part1/exercise/Exercise2.java]: Did you run test for this? it should we vice versa

**Review by vkovrovEPAM — COMMENTED** [review]
see inline comment

File: src/test/java/lambda/part1/exercise/Exercise2.java (hunk 1 above)
Comment (german1um) [reply src/test/java/lambda/part1/exercise/Exercise2.java]: i actually thought the value in test is incorrect, so i changed it, am gonna ask about it on lecture

Comment (german1um) [reply src/test/java/lambda/part1/exercise/Exercise2.java]: oh, i see my mistake now

**Review by ghost — APPROVED** [review]

**Pull request merged**